lxd/instance/drivers/driver_qemu.go

```diff
@@ -4025,20 +4025,58 @@ func (d *qemu) addDriveConfig(busAllocate busAllocator, bootIndexes map[string]i
 	}
 
 	// QMP uses two separate values for the cache.
-	directCache := true   // Bypass host cache, use O_DIRECT semantics by default.
-	noFlushCache := false // Don't ignore any flush requests for the device.
+	var directCache bool  // True to bypass host cache and use O_DIRECT semantics
+	var noFlushCache bool // True to ignore any flush requests for the device
+	// var writebackCache bool // True to complete writes once they are in the write page cache
```

Comment:
Would you mind updating the comment to add a link to https://gitlab.com/qemu-project/qemu/-/issues/3103 for us to reference in the future please.